docs/forms.md

```diff
@@ -0,0 +1,67 @@
+# Forms
+
+The Symfony Form component has a huge extension system, and allow us to customize it with extra behaviors.
+
+All this stuff is non-intrusive in your project and have to be activated throught bundle config, and a form option.
+
+## JSON extension
+
+By default, Form component can handle HttpKernel request, but only if data are sent in `GET`, or in `POST` (`PUT` / `PATCH`) with `application/x-www-form-urlencoded` or `multipart/form-data` encoded data.
+
+Not Json. With current microservices / REST / cloud mania. Mmmmkay !
+
+Json extension allows Form component to guess directly json into request content, and use it like row data.
```

Comment:
Pas trop compris la phrase.  
L'extension Json permet au `Form component` de deviner directement le json dans le contenu de la requête, et l'utiliser comme un `row data`?  
Le 'guess' est étrange ici. 

and use it like row data -> and uses it like row data